src/ValidatableExtensions/ValidatableExtensions.Strings.cs

```diff
@@ -200,4 +200,34 @@ public static partial class ValidatableExtensions
 
         return ref validatable;
     }
+
+    /// <summary>
+    /// Throws an exception if the string contains the given <paramref name="otherString"/>
+    /// Default <paramref name="comparisonType"/> is Ordinal.
+    /// </summary>
+    /// <remarks>
+    /// The default exception thrown is an <see cref="ArgumentException"/>.
+    /// </remarks>
+    public static ref readonly Validatable<string> IfContains(this in Validatable<string> validatable, string otherString, StringComparison comparisonType = StringComparison.Ordinal)
```

Comment:
This is great. It got me thinking that it's indeed more natural that each of the string validation methods will have the string comparison option (including `IfStrartsWith` etc) and having it defaulted to `Ordinal`